k8s-daemonset/README.md

```diff
@@ -24,42 +23,96 @@ docker push <helloworld image name>
 
 ## Deploying
 
-Deploy the hello and world python services, linkerd daemonset, and linkerd-viz
-to the default Kubernetes namespace.
+### Hello World
+
+Start by deploying the hello and the world apps by running:
 
 ```bash
-kubectl apply -f k8s/linkerd.yml
-kubectl apply -f k8s/linkerd-viz.yml
 kubectl apply -f k8s/hello-world.yml
 ```
 
-If you are running on Kubernetes 1.4 or later, you can alternatively run:
+Alternatively, if you are running on Kubernetes 1.4 or later, you can take
+advantage of some newer Kubernetes features, and run:
+
+```bash
+kubectl apply -f k8s/hello-world-1_4.yml
+```
+
+Either of these commands will create new Kubernetes Services and
+ReplicationControllers for both the hello app and the world app.
+
+### linkerd
+
+Next deploy linkerd, to route requests from the hello app to the world app.
```

Comment:
i think it's helpful to be super explicit that we're suggesting 3 distinct deployments here, something like

```
We will demonstrate three different deployment configurations, from simplest to most advanced.
...
#### DaemonSets
...
#### DaemonSets + TLS
...
#### DaemonSets + TLS + namerd
```